barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/circuit_simulator.hpp

```diff
@@ -189,6 +189,23 @@ class CircuitSimulatorBN254 {
 
     [[nodiscard]] bool check_circuit() const { return !_failed; }
 
+    size_t create_ROM_array([[maybe_unused]] const size_t array_size) { return {}; }
```

Comment:
had to add these since cycle group takes in CircuitSimulator now, because its being used in the ultra RecursiveVerifier